core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -182,7 +188,7 @@ public TranslationResult translateString(String sourceString, String sourceLangu
         	translatedString = String.format("%s_%s_%s", sourceLanguage, sourceString, targetLanguage);
         }else {
         	// Using Pseudo translation here using accented characters
-        	translatedString = bootstrapTmsService.getAccentedText(sourceString);	
+        	// translatedString = bootstrapTmsService.getAccentedText(sourceString);
```

Comment:
If you want to disable the pseudotranslation option, maybe better to just remove this conditional entirely?  (Or leave it intact and uncomment..)